.github/workflows/suite-native_production_ci.yml

```diff
@@ -58,6 +58,12 @@ jobs:
           bundle install
           bundle exec fastlane android production
 
+      - name: Upload
+        uses: actions/upload-artifact@v3
+        with:
+          name: Upload Artifacts
```

Comment:
Would set name something like Trezor suite native APK or something

Reply:
Yeah, good point. Fixed in https://github.com/trezor/trezor-suite/pull/8343/commits/44d4be9f8842233b4c093948e990125d9668bd1b